queries.sql

```diff
@@ -8,3 +8,35 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg>10.5;
 SELECT * FROM animals WHERE neutered=true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+-- changing species to unspecified
+ BEGIN;
+ UPDATE animals SET species='unspecified';
+ ROLLBACK;
+
+--  Updating the animal species
+ BEGIN;
+ UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+ UPDATE animals SET species='pokemon' WHERE species IS NULL;
+ SELECT * FROM animals;
+COMMIT;
+
+-- Delete Records
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] according to the requirements you should check if the records are deleted before the rollback.